src/lambdas/mps-report/src/generateReport.ts

```diff
@@ -54,19 +54,58 @@ export default async (gateway: PostgresGateway) => {
   result.push([""])
   result.push(headers)
   for (let i = 0; i < rows.length; i = i + 1) {
-    const annotatedMsg: string = rows[i].annotated_msg.replace(/br7:/g, "").replace(/ds:/g, "")
-    let annotatedMsgObject = (xml2js(annotatedMsg, { compact: true }) as AnnotatedPNCUpdateDataset)?.AnnotatedPNCUpdateDataset?.PNCUpdateDataset
-    if(!annotatedMsgObject) {
-      annotatedMsgObject = (xml2js(annotatedMsg, { compact: true }) as AnnotatedHearingOutcome)
-    }
-    const offences = annotatedMsgObject.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.Offence
-    const hearingOutcomeCase = annotatedMsgObject.AnnotatedHearingOutcome.HearingOutcome.Case
+    if(new Date(rows[i].court_date) < new Date()){
```

Comment:
Would you not just add this as a clause in the SQL?

`WHERE date < TIME()` or something?

Reply:
saves instantiating a new date object for every record and keeps the code simpler

Reply:
yup, good point. will make le changes